src/platform_impl/macos/view.rs

```diff
@@ -77,6 +78,7 @@ pub fn new_view(ns_window: id) -> (IdRef, Weak<Mutex<CursorState>>) {
         is_key_down: false,
         modifiers: Default::default(),
         tracking_rect: None,
+        mouse_buttons_down: 0,
```

Comment:
Mouse events provide the `pressedMouseButtons` field. Have you considered using that instead of implementing tracking yourself?